cfme/tests/infrastructure/test_set_default_filter.py

```diff
@@ -25,7 +25,7 @@ def unset_default_host_filter():
     appliance.server.logout()
     appliance.server.login_admin()
     navigate_to(host_collection, 'All')
-    assert view.filters.navigation.currently_selected[0] == 'Status / Running (Default)'
+    assert view.filters.navigation.currently_selected == ['Status / Running (Default)']
```

Comment:
REQUIRED: hehe, you still compare lists when there should be comparison of strings

Reply:
Oh, Thanks @izapolsk! Earlier it was giving list of all element since wrong locator was there, now it is giving single string in list, you were expecting this right?